<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFinderGUI</class>
 <widget class="QWidget" name="UserFinderGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Finder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/RedditDownloaderIcon.png</normaloff>../Images/RedditDownloaderIcon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="progress_label">
     <property name="text">
      <string>Progress Label</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="stacked_widget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="view_users_button">
         <property name="text">
          <string>View Users</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QSplitter" name="splitter_one">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="2">
            <widget class="QToolButton" name="remove_subreddit_button">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="add_subreddit_button">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Subreddit Watchlist</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QListWidget" name="subreddit_watchlist_widget">
             <property name="whatsThis">
              <string>A list of subreddits which tops posts will be examined to find users.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Do Not Include List</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="remove_user_button">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="add_user_button">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QListWidget" name="user_blacklist_widget">
             <property name="whatsThis">
              <string>A list of users who even if in the top posters for the subreddit list, will be ignored.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QSpinBox" name="score_limit_spinbox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Only posts with a score above this number will be considered</string>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="filter_by_score_checkbox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>If checked, only posts above the designated score will be considered</string>
             </property>
             <property name="text">
              <string>Filter By Score</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="post_limit_spinbox">
             <property name="toolTip">
              <string>The number of top posts that will be searched</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="toolTip">
              <string>The number of top posts that are looked at to get users.</string>
             </property>
             <property name="text">
              <string>Post Limit:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="top_sort_group_box">
             <property name="title">
              <string>Top Sort Period</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QRadioButton" name="hour_radio">
                <property name="text">
                 <string>Hour</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="day_radio">
                <property name="text">
                 <string>Day</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="week_radio">
                <property name="text">
                 <string>Week</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="month_radio">
                <property name="text">
                 <string>Month</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QRadioButton" name="year_radio">
                <property name="text">
                 <string>Year</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QRadioButton" name="all_time_radio">
                <property name="text">
                 <string>All Time</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="run_button">
         <property name="text">
          <string>Run User Finder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="settings_button">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="2">
        <widget class="QSplitter" name="splitter_two">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListView" name="user_list_view"/>
         <widget class="QSplitter" name="splitter_three">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Karma:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="user_karma_label">
              <property name="text">
               <string>karma</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Last Post Date:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>User Since:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number of Posts:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="user_member_since_label">
              <property name="text">
               <string>user since</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="ClickableLabel" name="user_post_count_label">
              <property name="text">
               <string>Click For Post Count</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="user_last_post_date_label">
              <property name="text">
               <string>last post date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QListWidget" name="content_list_widget"/>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="user_finder_button">
         <property name="text">
          <string>User Finder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
